sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,1674 @@
+      structs or SYCL classes like `accessor`
+    - using `-fsycl-dead-args-optimization` (ON by default) can lead to failures
+    - `info::kernel::num_args` won't return the right result for free function
+      kernels
+- Added experimental ESIMD function `fma` which results in a guaranteed fused
+  multiply-add operation performed. intel/llvm#13366
+- Implemented revision 2 of
+  [`sycl_ext_oneapi_group_sort`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_oneapi_group_sort.asciidoc)
+  extension. intel/llvm14399 intel/llvm#14185 intel/llvm#13942 intel/llvm#13908
+  intel/llvm#14591
+
+
+### SYCLcompat library
+
+- Added support for 2-byte and 4-byte `memset` operations. intel/llvm#13409
+- Added `compare`, `compare_both`, `compare_mask` and their `unordered_*`
+  counterparts. intel/llvm#12998
+- Added APIs for performing arithmetic operations on 33-bit extended values. intel/llvm#13006
+- Added APIs for performing bitwise operations on 33-bit extended values. intel/llvm#13727
+- Added `device_count` and `get_device_id` utility APIs. intel/llvm#14013
+- Added experimental `launch` API overloads that accept sub-group size. intel/llvm#13767
+- Added `wait` and `wait_and_throw` free functions. intel/llvm#13029
+- Added vectorized comparison `extend_vcompare[2|4]` APIs. intel/llvm#14079
+- Added vectorized math `extend_v*2` APIs. intel/llvm#13953
+- Added vectorized math `extend_v*4` APIs. intel/llvm#14078
+- Added bitfield manipulation APIs `bfe_safe` and `bfi_safe`. intel/llvm#14006
+- Added dot-product accumulate APIs `dp4a`, `dp2a_lo` and `dp2a_hi`. intel/llvm#14032
+- Added `wait_and_free` API. intel/llvm#14015
+- Added `filter_device` and `list_devices` APIs. intel/llvm#14016
+- Added `funnelshift_*` APIs. intel/llvm#13825
+- Added `match_[any|all]_over_sub_group` APIs. intel/llvm#12973
+- Added API to manage kernel libraries loading/unloading. intel/llvm#13053 intel/llvm#13932
+- Added `cmul_add` API. intel/llvm#12969
+- Added experimental APIs for maksed operations over sub-groups (`select`, `shift`, etc.). intel/llvm#12972
+- Added various helper APIs: a mechanism to extract arguments from a kernel and
+  its kernel parameters; type casting helper for generic address -> queue
+  pointer; a wrapper to provide better support for logical groups; an enum to
+  list supported group types. intel/llvm#12970
+- Added wrappers/support to math functions `clamp`, `isnan`, `cbrt`, `min`,
+  `max`, `fmin_nan`, `fmax_nan`, `pow`, `relu`; wrappers are needed to support
+  variety of combinations of argument types compared to `sycl::` counterparts
+  of those functions. intel/llvm#13005
+- Added `SYCLCOMPT_CHECK_ERROR` macro which is an error handling utility for
+  expressions that throw exceptions.
+- Added `image1d_max`, `image2d_max` and `image3d_max` device info getters
+  and setters. intel/llvm#13973
```

Comment:
```suggestion
- Added `image1d_max`, `image2d_max` and `image3d_max` `device_info` getters
  and setters. intel/llvm#13973
```

Reply:
Applied in cf3fb5e9e9be153e83c3e058e053012aa1b3a5f0